js/components/TodoList.js

```diff
@@ -0,0 +1,196 @@
+
+			const {action} = event.target.dataset;
+
+			if (action === 'onEditingContents') {
+				event.stopPropagation();
+				event.target.closest('li').classList.add('editing');
+				return;
+			}
+
+			if (action === 'onEditingPriority') {
+				event.stopPropagation();
+				const $todoItem = event.target.closest('li');
+				const todoItemId = $todoItem.dataset.todoItemId;
+				this.state.todoItems.find(
+					(todoItem) => todoItem._id === todoItemId,
+				).onEditingPriority = true;
+
+				this.setState({todoItems: this.state.todoItems});
+			}
+		});
+
+		this.$el.addEventListener('keypress', (event) => {
+			if (event.key !== 'Enter' && event.key !== 'Escape') {
+				return;
+			}
+
+			const {action} = event.target.dataset;
+			const $todoItem = event.target.closest('li');
+			const todoItemId = $todoItem.dataset.todoItemId;
+
+			if (action === 'edit') {
+				if (event.key === 'Enter') {
+					editTodoItemContents(todoItemId, event.target.value);
+					return;
+				}
+				if (event.key === 'Escape') {
+					event.target.value = this.state.todoItems.find(
+						(todoItem) => todoItem._id === todoItemId,
+					).contents;
+					$todoItem.classList.remove('editing');
+					return;
+				}
+			}
+		});
+
+		this.$el.addEventListener('change', (event) => {
+
+			const {action} = event.target.dataset;
+
+			if (action === 'changePriority') {
+				const priority = event.target.value;
+				const $todoItem = event.target.closest('li');
+				const todoItemId = $todoItem.dataset.todoItemId;
+
+				editTodoItemPriority(todoItemId, priority);
+			}
+		});
+	};
+
+	const makeTodoItemLoadingTemplate = function () {
+
+		return `
+			<li>
+				<div class="view">
+				<label class="label">
+					<div class="animated-background">
+					<div class="skel-mask-container">
+						<div class="skel-mask"></div>
+					</div>
+					</div>
+				</label>
+				</div>
+			</li>
+		`;
+	};
+
+	const makeTodoItemTemplate = function (todoItem) {
+
+		const {_id, contents, priority, isCompleted, onEditingPriority} = todoItem;
+		return `
+			<li class="${isCompleted ? 'completed' : ''}" data-todo-item-id="${_id}">
+				<div class="view">
+					<input class="toggle" type="checkbox" data-action="toggle" ${isCompleted ? 'checked' : ''} />
+					<label class="label" data-action="onEditingContents">
+						<div class="chip-container">
+							${makePriorityTemplate(priority, onEditingPriority)}
+						</div>
+						${contents}
+					</label>
+					<button class="destroy" data-action="delete"></button>
+				</div>
+				<input class="edit" data-action="edit" value="${contents}" />
+			</li>
+		`;
+	};
+
+	const sortByPriority = (todoItemA, todoItemB) => {
+
+		const {priority: priorityA} = todoItemA;
+		const {priority: priorityB} = todoItemB;
+
+		if (PRIORITY_TYPE[priorityA].order > PRIORITY_TYPE[priorityB].order) {
+			return 1;
+		}
+
+		if (PRIORITY_TYPE[priorityA].order < PRIORITY_TYPE[priorityB].order) {
+			return -1;
+		}
+
+		return 0;
```

Comment:
```suggestion
		return PRIORITY_TYPE[priorityA].order - PRIORITY_TYPE[priorityB].order;
```
1과 -1이 아닌 양수값, 음수값 (그리고 0)으로 정렬이 결정되니, 수식을 바로 리턴하는건 어떨까요?